src/heuristicblocking.js

```diff
@@ -211,6 +211,13 @@ HeuristicBlocker.prototype = {
       snitch_map.setItem(tracker_origin, firstParties);
     }
 
+    var action_map = this.storage.getBadgerStorageObject('action_map');
+    // If tracker_origin is not yet in action_map, add it with action `constants.ALLOW`
+    if (!action_map.hasItem(tracker_origin)) {
```

Comment:
Right, that makes sense. I was rushing out the door and probably should've taken a moment to read the code more carefully!

I suppose we don't need the check anymore for performance reasons; it's just weirdly structured code though so maybe the check helps with readability. I'll let you confirm or deny that, though :)